src/getPostcssResult.js

```diff
@@ -43,13 +43,21 @@ export default function (
       } else if (syntax === "sugarss" || !syntax && fileExtension === ".sss") {
```

Comment:
If we don't move the module resolution logic into here the above can just be `if (!syntax && fileExtension === ".sss")`